installer/manifests/common/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v2
```

Comment:
Isn't it weird to have the common chart in multiple places? (helm/jmeter service)

Reply:
The common chart isn't published anywhere as far as I can tell, therefore to be able to use it in the build pipeline it has to be duplicated in a few places. But for now I removed it completely